src/components/protocol_handler/include/protocol_handler/protocol_handler_impl.h

```diff
@@ -160,6 +160,18 @@ struct TransportDescription {
 typedef std::map<std::string, TransportDescription> TransportTypes;
 }  // namespace impl
 
+#ifdef BUILD_TESTS
+/**
+ * Function return packet data as std::string.
+ * If packet data is not printable return error message
+ */
+std::string ConvertPacketDataToString(const uint8_t* data,
```

Comment:
@mkorniichuk dowe rely need to add this add this functionwith BUILD_TESTS flag? It is rather bad practive to extend class for testing

Reply:
@LuxoftAKutsan A function ConvertPacketDataToString as well as get_hash_id and set_hash_id are functions, not a class members and are used only in protocol_handler_impl.cc. Their declaration can be removed from the header file at all, but in this case, all tests for these functions have to be removed as well. And, I suppose, include source file is even worse.